DataStructure/src/main/list/ResizableArray.java

```diff
@@ -54,21 +64,22 @@ public void add(int num){
      *  or if the given index is greater than or equals to size.
      */
     public int get(int index) {
-        if (index < 0 || index >= writeIndex){ throw new ArrayIndexOutOfBoundsException(); }
+        if (index < 0 || index >= writeIndex ){
+            throw new ArrayIndexOutOfBoundsException();
+        }
         return list[index];
     }
 
     /**
      * Sets given element at the specified index.
-     *
      * @param index
      * @param num
      * @throws ArrayIndexOutOfBoundsException
      *  If the given index is less than 0
      *  or if the given index is greater than or equals to size.
      */
     public void set(int index, int num)  {
-        if (index < 0 || index >= writeIndex) {
+        if (index < 0 || index >= writeIndex || isEmpty()){
```

Comment:
Creating code review with isEmpty method here = Unprofessionalism.

Reply:
it was not in my main code I don't know why